lib/circuitbox/circuit_breaker.rb

```diff
@@ -29,9 +28,8 @@ def initialize(service, options = {})
       @execution_timer = options.fetch(:execution_timer) { Circuitbox.default_timer }
       @notifier = options.fetch(:notifier) { Circuitbox.default_notifier }
 
-      @exceptions = options.fetch(:exceptions) { [] }
+      @exceptions = options.fetch(:exceptions)
```

Comment:
Maybe we need a short and concise "upgrade guide" for migrating from 1.x to 2.x.